flooder/src/main.rs

```diff
@@ -78,142 +79,167 @@ fn main() -> Result<(), anyhow::Error> {
+                        "estimating required amount: {}ms",
+                        time_stats.elapsed().as_millis()
+                    );
+
+                    let total_amount = estimate_amount(
+                        &connection,
+                        &account,
+                        source_account_nonce,
+                        transfer_amount,
+                    );
+
+                    info!("amount estimated: {}ms", time_stats.elapsed().as_millis());
+
+                    assert!(
+                        get_funds(&connection, &source_account_id)
+                            .ge(&(total_amount * total_users as u128)),
+                        "Account is too poor"
+                    );
+
+                    info!(
+                        "initializing accounts: {}ms",
+                        time_stats.elapsed().as_millis()
+                    );
+
+                    initialize_accounts(
+                        &connection,
+                        &account,
+                        source_account_nonce,
+                        &accounts,
+                        total_amount,
+                    );
+
+                    info!(
+                        "accounts initialized: {}ms",
+                        time_stats.elapsed().as_millis()
+                    );
+
+                    debug!("all accounts have received funds");
+                }
+
+                info!(
+                    "initializing nonces: {}ms",
+                    time_stats.elapsed().as_millis()
+                );
+                debug!("all accounts have received funds");
+                let nonces: Vec<_> = match config.download_nonces {
+                    false => repeat(0).take(accounts.len()).collect(),
+                    true => accounts
+                        .par_iter()
+                        .map(|account| get_nonce(&connection, &AccountId::from(account.public())))
+                        .collect(),
+                };
+                info!("nonces initialized: {}ms", time_stats.elapsed().as_millis());
+                let receiver = accounts
+                    .first()
+                    .expect(
+                        "we should be some accounts available for this test, but the list is empty",
+                    )
+                    .clone();
+
+                info!(
+                    "signing transactions: {}ms",
+                    time_stats.elapsed().as_millis()
+                );
+
+                let txs: Vec<_> = sign_transactions(
+                    connection.clone(),
+                    receiver,
+                    accounts.into_par_iter().zip(nonces),
+                    transfer_amount,
+                )
+                .collect();
+
+                info!(
+                    "transactions signed: {}ms",
+                    time_stats.elapsed().as_millis()
+                );
+
+
+                if store_txs {
+                    zip_and_store_txs(txs);
```

Comment:
probably should be `zip_and_store_txs(txs, config.tx_store_path)`

Reply:
Fixed.